src/frontend/src/icp-eth/services/eth.services.ts

```diff
@@ -6,6 +6,7 @@ import { alchemyProviders } from '$eth/providers/alchemy.providers';
 import { infuraCkETHProviders } from '$eth/providers/infura-cketh.providers';
 import { isSupportedEthTokenId } from '$eth/utils/eth.utils';
 
+import type { Erc20Token } from '$eth/types/erc20';
```

Comment:
NTH: I guess this can be moved one line up, line 8 is empty

Reply:
right!